src/app/components/HelloWorld/HelloWorld.scss

```diff
@@ -0,0 +1,14 @@
+h3 {
+  margin: 40px 0 0;
+}
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+li {
+  display: inline-block;
+  margin: 0 10px;
+}
+a {
+  color: #42b983;
```

Comment:
What kind of color is this? Does it make sense to make it available globally by sass variable?

Reply:
It's an example. no need for that.